Sources/SwagGen/GenerateCommand.swift

```diff
@@ -1,4 +1,7 @@
 import Foundation
+#if canImport(FoundationNetworking)
+import FoundationNetworking
```

Comment:
Do you really need this `import`? There is not such line at https://github.com/yonaskolb/SwagGen/blob/master/Sources/SwagGen/GenerateCommand.swift.

Reply:
@romanpodymov it is sadly required yes.

Yonas's repository either is not working on Linux due to the same issue.

Swift 5.1 moved all of Foundation's networking-related code to a separate module, which you have to include separately with import FoundationNetworking

Cf the following:
https://github.com/apple/swift-corelibs-foundation/pull/2289